azplugins/test-py/test_bond_fene.py

```diff
@@ -52,16 +52,6 @@ def test_set_missing_r0(self):
         self.assertRaises(RuntimeError, fene.update_coeffs)
 
      # test coefficients = 0
-    def test_set_zero_epsilon(self):
-        fene = azplugins.bond.fene()
-        fene.bond_coeff.set('bond', epsilon=0, sigma=1.0, k=30,r0=1.5)
-        self.assertRaises(ValueError, fene.update_coeffs)
-
-    def test_set_zero_sigma(self):
-        fene = azplugins.bond.fene()
-        fene.bond_coeff.set('bond', epsilon=1.0, sigma=0.0, k=30, r0=1.5)
-        self.assertRaises(ValueError, fene.update_coeffs)
-
     def test_set_zero_k(self):
```

Comment:
Suggest to remove (see above)

Reply:
Should I add a test when k=0?

Reply:
It is not necessary. Unlike for the LJ parameters, there is no place where K could be used to divide by zero.

Reply:
Oh, I did not see you message and added it and asked for a review. But no worries, I will remove it